markdown/summary.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+output: 
+  word_document:
+    reference_docx: "summary-template.docx"
+---
+
+```{r, echo=FALSE, message=FALSE, warning=FALSE,error=FALSE}
+
+#Create a pseudo chart
+library(ggplot2)
+
+Year <- c(rep("2014", times = 8), rep("2015", times = 8))
+Month <- rep(c(4, 5, 6, 7, 8, 9, 10, 11), times = 2)
+Count <- c(800, 820, 840, 860, 880, 1000, 1100, 1200, 
+           800, 840, 860, 880, 1000, 1100, 1200, 1300)
+
+ChartData <- data.frame(Year, Month, Count)
+
+chart <- ggplot(data = ChartData, aes(x = Month, y = Count, group = Year, color = Year)) +
+  geom_line(size = 1) + 
+  geom_point(size = 3) +
+  theme(plot.title = element_text(hjust = 0.5, face = "bold", size = 11),
```

Comment:
I noticed the chart title is also in purple. I found the hex colour for it. So we need to add an argument for colour:

plot.title = element_text(hjust = 0.5, face = "bold", size = 11, colour = "#6C2383")

I will add this to the National-Stats-Template as well.

Reply:
Thanks Tina - I'll remember this for when we add charts back in